examples/etcd/start-etcd-insecure.sh

```diff
@@ -6,8 +6,8 @@
 #
 # NOTE: this file is also used to run etcd tests.
 #
-HERE=$(readlink -f $0)
-cd "$(dirname $HERE)" || exit
+HERE=$(readlink -f "$0")
+cd "$(dirname "$HERE")" || exit
```

Comment:
```suggestion
cd $(dirname "$HERE") || exit
```
I think this will break otherwise because the external part is already wrapped in `"`. At any rate, it's having the exact opposite effect of what was intended (i.e. wrap the variable expansions in `" "`)